Plots/MapProjections/NCL_proj_3_lg.py

```diff
@@ -0,0 +1,80 @@
+
+"""
+NCL_proj_3_lg.py
+================
+
+This script illustrates the following concepts:
+   - Drawing filled contours over an orthographic map
+   - Changing the center latitude and longitude for an orthographic projection
+   - Turning off map fill
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/proj_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/proj_3_lg.png
+"""
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
```

Comment:
I suggested to remove `newcmp = gvcmaps.gui_default` below, so this should be removed as well.